pkg/templates/templates_generated.go

```diff
@@ -3126,27 +3112,8 @@ write_files:
   permissions: "0644"
   owner: root
   content: |
-    subreaper = false
-    oom_score = 0
-    [plugins.cri]
-    sandbox_image = "{{GetPodInfraContainerSpec}}"
-    [plugins.cri.containerd.untrusted_workload_runtime]
-    runtime_type = "io.containerd.runtime.v1.linux"
-    {{if IsKataContainerRuntime }}
-    runtime_engine = "/usr/bin/kata-runtime"
-    {{else}}
-    runtime_engine = "/usr/local/sbin/runc"
-    {{end}}
-    [plugins.cri.containerd.default_runtime]
-    runtime_type = "io.containerd.runtime.v1.linux"
-    {{if IsKataContainerRuntime }}
```

Comment:
can we not change the template in place?

Reply:
how do you mean?

Reply:
ok. so this is like the same change to the docker config, we only need to add a root dir. I have comment around the docker change. I will close this and we can use that one for discussion